Makefile

```diff
@@ -146,6 +146,11 @@ build:
 	@echo LDFLAGS=$(LDFLAGS)
 	go build -o build/_output/bin/baremetal-operator cmd/manager/main.go
 
+.PHONY: tools
+tools:
+	@echo LDFLAGS=$(LDFLAGS)
+	go build -o build/_output/bin/make-introspection cmd/make-introspection/main.go
```

Comment:
I'm not sure about the name of the command. Maybe something closer to the API field name, like get-hardware-details? It's not worth holding up the initial version, but something to think about.

Reply:
I like `get-hardware-details`.